rdkit/Chem/FeatFinderCLI.py

```diff
@@ -37,79 +26,85 @@ def GetAtomFeatInfo(factory, mol):
   feats = factory.GetFeaturesForMol(mol)
   for feat in feats:
     ids = feat.GetAtomIds()
-    for id in ids:
-      if res[id] is None:
-        res[id] = []
-      res[id].append("%s-%s" % (feat.GetFamily(), feat.GetType()))
+    feature = "%s-%s" % (feat.GetFamily(), feat.GetType())
+    for id_ in ids:
+      if res[id_] is None:
+        res[id_] = []
+      res[id_].append(feature)
   return res
 
 
-if __name__ == '__main__':
+def initParser():
+  """ Initialize the parser """
+  parser = argparse.ArgumentParser(description='Create aligned depiction', epilog=_splashMessage,
+                                   formatter_class=argparse.RawDescriptionHelpFormatter)
+  parser.add_argument('-r', dest='reverseIt', default=False, action='store_true')
+  parser.add_argument('-n', dest='maxLines', default=-1, help=argparse.SUPPRESS, type=int)
+  parser.add_argument('fdefFilename', type=existingFile)
+  parser.add_argument('smilesFilename', type=existingFile,
+                      help='The smiles file should have SMILES in the first column')
+  return parser
 
-  def Usage():
-    message = """
-    Usage: FeatFinderCLI [-r] <fdefFilename> <smilesFilename>
-    
-    NOTE:
-      - the smiles file should have SMILES in the first column
-
-    """
-    print(message, file=sys.stderr)
-
-  import getopt
-  args, extras = getopt.getopt(sys.argv[1:], 'r')
-  reverseIt = False
-  for arg, val in args:
-    if arg == '-r':
-      reverseIt = True
-
-  if len(extras) < 2:
-    Usage()
-    sys.exit(-1)
-  print(_splashMessage, file=sys.stderr)
-  fdefFilename = extras[0]
-  if not os.path.exists(fdefFilename):
-    logger.error("Fdef file %s does not exist." % fdefFilename)
-    sys.exit(-1)
-  try:
-    factory = ChemicalFeatures.BuildFeatureFactory(fdefFilename)
-  except Exception:
-    logger.error("Could not parse Fdef file %s." % fdefFilename, exc_info=True)
-    sys.exit(-1)
 
-  smilesFilename = extras[1]
-  if not os.path.exists(smilesFilename):
-    logger.error("Smiles file %s does not exist." % smilesFilename)
-    sys.exit(-1)
+_splashMessage = """
+-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
```

Comment:
This is a horrible historical artifact.
Please change it to something neutral and correct like:

```
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
FeatFinderCLI
  Part of the RDKit (http://www.rdkit.org)
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
```